### ISSUE TITLE
About Result

### ISSUE THREAD

**Issue by openchao** [Jaewoong-Lee/sigir_2023_uCTRL#3]
Your work is very interesting. But when I reappeared, I found that your results on Beauty and Gowalla were lower than your baseline (DirectAU). For example, on Beauty, DirectAU(R@20:14.07%) and uCTRL(R@20:4.64%). Is this your super reference given by mistake? Or I didn't run your code correctly.

**Comment by Jaewoong-Lee** [comment]
Thank you for your interest in my research. I apologize for not providing instructions for each dataset. For Gowalla, would you consider training the model using the following hyperparameters?

--eval=unbiased
--gamma1 2
--gamma2 30
--weight_decay 0

Here, 'eval=unbiased' is used to find the optimal epoch in an unbiased mode (with unbiased evaluation) since the validation set is biased.

For Beauty, it might be necessary to conduct a hyperparameter search.
